packages/govuk-frontend/src/govuk/components/label/_index.scss

```diff
@@ -9,24 +9,27 @@
   }
 
   // Modifiers that make labels look more like their equivalent headings
+  .govuk-label--xl,
+  .govuk-label--l,
+  .govuk-label--m {
+    @include govuk-typography-weight-bold;
+    margin-bottom: govuk-spacing(3);
+  }
 
   .govuk-label--xl {
-    @include govuk-font($size: 48, $weight: bold);
-    margin-bottom: govuk-spacing(3);
+    @include govuk-typography-responsive($size: 48);
   }
 
   .govuk-label--l {
-    @include govuk-font($size: 36, $weight: bold);
-    margin-bottom: govuk-spacing(3);
+    @include govuk-typography-responsive($size: 36);
   }
 
   .govuk-label--m {
-    @include govuk-font($size: 24, $weight: bold);
-    margin-bottom: govuk-spacing(2);
+    @include govuk-typography-responsive($size: 24);
   }
 
   .govuk-label--s {
-    @include govuk-font($size: 19, $weight: bold);
+    @include govuk-typography-weight-bold;
```

Comment:
Should the whole rule be removed and put `govuk-typography-weight-bold` in `.govuk-label`?

Means you can remove it further up too

```patch
  .govuk-label--xl,
  .govuk-label--l,
  .govuk-label--m {
-   @include govuk-typography-weight-bold;
    margin-bottom: govuk-spacing(3);
  }
```  

```suggestion
```

Reply:
Default labels aren't bold so I think this is the best we can do right now

Reply:
Ahh you learn something new every day (just found [Labels legends and headings](https://govuk-frontend-review.herokuapp.com/examples/labels-legends-and-headings))

Thanks for checking